.eslintignore

```diff
@@ -0,0 +1,2 @@
+react-app-env.d.ts
```

Comment:
In `.eslintignore` können wir alle Dateien definieren, die wir nicht linten wollen. Das hier ist eine Datei die automatisch von create-react-app erstellt wird und die wir manuell gar nicht ändern wollen. Insofern ignorieren wir die hier.